SwiftHelpers/SHUIColorExtension.swift

```diff
@@ -84,6 +84,27 @@ public extension UIColor {
         return UIColor(red: CGFloat(r), green: CGFloat(g), blue: CGFloat(b), alpha: 1.0)
     }
     
+    public func matchWithColor(color: UIColor, withTolerance tolerance: CGFloat = 0.0) -> Bool {
+        
+        var r1 : CGFloat = 0
+        var g1 : CGFloat = 0
+        var b1 : CGFloat = 0
+        var a1 : CGFloat = 0
+        var r2 : CGFloat = 0
+        var g2 : CGFloat = 0
+        var b2 : CGFloat = 0
+        var a2 : CGFloat = 0
+        
+        self.getRed(&r1, green: &g1, blue: &b1, alpha: &a1)
+        color.getRed(&r2, green: &g2, blue: &b2, alpha: &a2)
```

Comment:
I thought @dmiotti already added a `.rgbComponents` variable on UIColor, might be nice to use this instead